egs/fisher_swbd/s5/local/chain/run_tdnn_7b.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
add a comment saying that this is based on script TDNN-7b in the swbd recipe, otherwise the version number does not make sense.